### ISSUE TITLE
Outdated config options throw deprecation warnings 

### ISSUE THREAD

**Issue by raphaelokon** [chrisprice/grunt-rollup#39]
It seems like the internal `grunt-rollup` options are outdated. We are getting some sorts of this here in the logs …
```
The following options have been renamed — please update your config: interop -> output.interop
Unknown option found: moduleId. Allowed keys: input, legacy, treeshake, acorn […]
``` 

I had a super-quick look into the `rollup.js` task file and it seems like https://github.com/chrisprice/grunt-rollup/blob/master/tasks/rollup.js#L20-L44 isn't par with https://rollupjs.org/guide/en#javascript-api

Probably @lukastaegert or @rich-harris could share some light on this.

Thanks

**Comment by fyrkant** [comment]
I created a PR (#40) that I think will fix this.

**Comment by plegner** [comment]
Any update on when this will be done?

**Comment by multicam** [comment]
Same here :)

**Comment by GMartigny** [comment]
This is fixed by the [v10.0.0](https://github.com/chrisprice/grunt-rollup/releases/tag/v10.0.0) release. :tada:
